test/ts/node-src/node.spec.ts

```diff
@@ -4,5 +4,7 @@ process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
 process.env.DISABLE_CORS_TESTS = "true";
 // Disable the WebSocket-based tests as they don't apply for Node environments
 process.env.DISABLE_WEBSOCKET_TESTS = "true";
+// Although this will be set by Travis CI; it won't be set locally.
+process.env.BROWSER = "nodejs";
```

Comment:
As the comment suggest, this is required when running the tests locally; this was not an issue before because the `node-http` transport was automatically detected by default; however now `testRpcCombinations.ts` sniffs for this environment variable and configures the `NodeHttpTransport` accordingly.